nsv13/code/game/machinery/iff_console.dm

```diff
@@ -33,6 +33,9 @@ If someone hacks it, you can always rebuild it.
 /obj/machinery/computer/iff_console/LateInitialize()
 	. = ..()
 	var/obj/structure/overmap/OM = get_overmap()
+	if(OM?.hammerlocked == TRUE) //In case the console gets destroyed and needs to be rebuilt
+		start_emagged = TRUE
+		radio_channel = RADIO_CHANNEL_SYNDICATE
```

Comment:
```suggestion
		if(OM.faction == "Syndicate")
			radio_channel = RADIO_CHANNEL_SYNDICATE
		else if(OM.faction == "Pirate")
			radio_channel = RADIO_CHANNEL_PIRATE
```
Because the radio is still faction dependant 🙃